Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mv bws /usr/local/bin
 RUN rm -r /tmp/
 
 # Install the dependencies
-RUN pip install --no-cache-dir -r requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip install -r requirements.txt
```

Comment:
Unsure if this is a good idea or not but haven't seen it used before

Reply:
This create a pip cache that is stored outside of the docker container, speeding up rebuilds 
https://pythonspeed.com/articles/docker-cache-pip-downloads/

Reply:
Speeds up rebuilds that require a pip install by caching the dependencies

https://pythonspeed.com/articles/docker-cache-pip-downloads/

Reply:
Just looking at random repo and came across this. If you want a faster build, you can always setup CI pipeline with GH Action and store it on PKG registry, then fetch it on the server.